WordPress/Classes/Services/MediaRepository.swift

```diff
@@ -59,4 +59,31 @@ final class MediaRepository {
 
         return nil
     }
+
+}
+
+extension MediaServiceRemote {
+
+    func getMediaLibraryCount(forMediaTypes types: [MediaType]) async throws -> Int {
```

Comment:
Did you consider dropping the `MediaTypes` given the information is already part of the type?

```suggestion
    func getMediaLibraryCount(for types: [MediaType]) async throws -> Int {
```